vnext/Microsoft.ReactNative/Microsoft.ReactNative.vcxproj

```diff
@@ -191,6 +191,9 @@
     <ClInclude Include="DynamicWriter.h">
       <DependentUpon>IJSValueWriter.idl</DependentUpon>
     </ClInclude>
+    <ClInclude Include="JSDispatcherWriter.h">
+      <DependentUpon>IJSValueWriter.idl</DependentUpon>
```

Comment:
looks like you forgot to add the IDL files?

Reply:
No, there are no new IDL files. Please note that we use the existing IJSValueWriter.idl, we just add another implementation of that public interface.